scipy/stats/_multivariate.py

```diff
@@ -960,6 +964,36 @@ def rvs(self, mean=None, rowcov=1, colcov=1, size=1, random_state=None):
             out = out.reshape(mean.shape)
         return out
 
+    def equivalent_multivariate_normal(self, mean=None, rowcov=1, colcov=1,
+                                       random_state=None):
```

Comment:
(It might have been me, asking for a line or two in the docs and being unclear.)

The documentation tweaks are good, would you submit those? 
The easiest for us would be if you change the PR in place and force-push. Or a separate PR, whichever works for you.